extras/videoDrivers/SDL2/VideoSDL2.cpp

```diff
@@ -25,6 +26,9 @@
 #    include <windows.h> // Avoid "Windows headers require the default packing option" due to SDL2
 #    include <SDL_syswm.h>
 #endif // _WIN32
+#if RTTR_OGL_GL4ES == 1
```

Comment:
`RTTR_OGL_GL4ES` should be defined only if available, so `#ifdef RTTR_OGL_GL4ES` should be used?